eodatasets3/assemble.py

```diff
@@ -846,12 +857,52 @@ def write_thumbnail(
             percentile_stretch=percentile_stretch,
             input_geobox=grid,
         )
-        self._checksum.add_file(thumb_path)
 
-        accessory_name = "thumbnail"
-        if kind:
-            accessory_name += f":{kind}"
-        self.add_accessory_file(accessory_name, thumb_path)
+        self._document_thumbnail(thumb_path, kind)
+
+    def write_thumbnail_singleband(
+        self,
+        measurement: str,
+        thumb_path,
```

Comment:
This argument may have been for debugging? (it's never used, being overridden in the first line. And the paired `create_thumbnail` function doesn't have it either)